dashboards-observability/public/components/application_analytics/components/app_table.tsx

```diff
@@ -0,0 +1,246 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+/*
```

Comment:
Are we supposed to keep any Amazon copyright notices in our files anymore? I thought we were supposed to remove them and only have the OpenSearch license headers

Reply:
the license headers should just be

```
/*
 * Copyright OpenSearch Contributors
 * SPDX-License-Identifier: Apache-2.0
 */
```

Reply:
Oh, you're right. I wrote this before we changed all the copyright headers. On it.